presto-main/src/main/java/com/facebook/presto/sql/analyzer/MaterializedViewInformationExtractor.java

```diff
@@ -50,6 +54,9 @@ protected Void visitQuerySpecification(QuerySpecification node, Void context)
             throw new SemanticException(NOT_SUPPORTED, node, "Having clause is not supported in query optimizer");
         }
         materializedViewInfo.setWhereClause(node.getWhere());
+        if (node.getFrom().isPresent() && node.getFrom().get() instanceof AliasedRelation) {
+            materializedViewInfo.setTableAliased(true);
```

Comment:
We probably don't need to store table alias information, we can remove it safely if present.